src/routers/resource_router.py

```diff
@@ -485,13 +486,39 @@ def register_resource(
 
         return register_resource
 
-    def create_resource(self, session: Session, resource_create_instance: SQLModel):
+    def create_resource(
+        self,
+        session: Session,
+        resource_create_instance: SQLModel,
+        user: KeycloakUser = Depends(get_user_or_raise),
```

Comment:
```suggestion
        user: KeycloakUser,
```
I believe this function is never registered as an API endpoint, do this dependency injection doesn't work (which is fine, you explicitly pass the user anyway).

Reply:
yes, the dependency injection is making a difference here.